.identity/04_github_environment.tf

```diff
@@ -0,0 +1,32 @@
+resource "github_repository_environment" "prod_ci" {
```

Comment:
I'd prefer to add an `opex` suffix somewhere, like [this](https://github.com/pagopa/io-sign/blob/main/.identity/github_environment_opex.tf)